Components/Lists/list.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lists</title>
+    <link rel="stylesheet" href="../../Docs/doc.css">
+    <link rel="stylesheet" href="list.css">
+</head>
+<body>
+    <nav class='nav'>
+        <div >
+            <a class="nav-header" href="">Ozone UI</a>
+        </div>
+        <div class="nav-links">
+            <a class='nav-item' href="https://twitter.com/reddygov16" target="_blank">
+                <img class="logo" src="../../Assets/SVG/twitter.png" alt="twitter">
+            </a>
+            <a class='nav-item' href="https://github.com/govardhanreddyc79" target="_blank">
+                <img class='logo' src="../../Assets/SVG/github.png" alt="github">
+            </a>
+        </div>
+    </nav>
+    <main class="main-section">
+        <aside class="sidebar flex-column">
+            <a class ='sidebar-header' href="">Getting Started</a>
+            <a class ='sidebar-item hover-effect' href="">Installation Guide</a>
+    
+            <a class ='sidebar-header' href="">Components</a>
+            <a class ='sidebar-item hover-effect' href="">Alert</a>
+            <a class ='sidebar-item hover-effect' href="">Avtar</a>
+            <a class ='sidebar-item hover-effect' href="">Badge</a>
+            <a class ='sidebar-item hover-effect' href="">Button</a>
+            <a class ='sidebar-item hover-effect' href="">Card</a>
+            <a class ='sidebar-item hover-effect' href="">Image</a>
+            <a class ='sidebar-item hover-effect' href="">Input</a>
+            <a class ='sidebar-item link-active hover-effect' href="">Lists</a>
+            <a class ='sidebar-item hover-effect' href="">Text Utilities</a>
+            <a class ='sidebar-item hover-effect' href="">Toasts</a>
```

Comment:
Here too, href it is not pointing to anything.  Are you planning to provide the references this at the end? Let me know your thoughts on this.

Reply:
will add those links, Thanks